app/serializers/fikas/index_serializer.rb

```diff
@@ -1,4 +1,4 @@
 class Fikas::IndexSerializer < ActiveModel::Serializer
-  attributes :id, :start_date, :end_date, :similarity
+  attributes :id, :formated_start_date, :formated_end_date, :similarity
```

Comment:
why have you decided to change the name to `formated...`?

Reply:
Thanks for the steer, we've refactored to override the definition of `:start:date` in the serializer as opposed to defining a new attribute in the model